src/easylink/pipeline_graph.py

```diff
@@ -46,9 +46,14 @@ def update_slot_filepaths(self, config: Config) -> None:
             "pipeline_graph_input_data", data=True
         ):
             for edge_idx in self[source][sink]:
-                self[source][sink][edge_idx]["filepaths"] = [
-                    str(config.input_data[edge_attrs["output_slot"].name])
-                ]
+                if edge_attrs["output_slot"].name == "all":
```

Comment:
It seems like the number of framework key strings is growing ("all", "output_slot", "filepaths", etc). Should they be encapsulated soewhere?

Reply:
What do you have in mind specifically? I could see a few ways of doing this.
One is just to define the strings as constants somewhere, which is maybe the right thing to do for `"all"`

Networkx stores edge attributes as a dictionary, which is why I'm using keys and not defining some attribute like "edge.output_slot", though there is a relationship between the edge attributes defined in `graph_components.py` and the keys in the actual graph dictionary. Though we could possibly subclass the MultiDiGraph in such a way that we can define these attributes separately somehow. I've thought about doing that but it would be a refactoring task.